<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CQUPT WMS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icons/CQUPT.ico</normaloff>:/icons/CQUPT.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(76, 76, 76);
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="GraphicsWidget" name="graphicsWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="viewMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="viewSettings_dw">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>1000</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(250, 250, 250);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="ViewSettingsWidget" name="viewSettings_w"/>
  </widget>
  <widget class="QDockWidget" name="minimap_dw">
   <property name="windowTitle">
    <string>Minimap</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="MinimapView" name="minimap"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Channel Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ViewSettingsWidget</class>
   <extends>QWidget</extends>
   <header>ViewSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicsWidget</class>
   <extends>QWidget</extends>
   <header>GraphicsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MinimapView</class>
   <extends>QWidget</extends>
   <header>MinimapView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
